pkg/arm/data/azuredeploy.json

```diff
@@ -163,6 +163,20 @@
                 ]
             }
         },
+        {
+            "apiVersion": "2014-10-01-preview",
+            "name": "[guid(concat('Microsoft.Compute/virtualMachineScaleSets/', 'ss-{{ $app.Name }}', 'vmidentity'))]",
+            "type": "Microsoft.Authorization/roleAssignments",
+            "properties": {
+{{- if or (eq $app.Role "master") (eq $app.Role "infra") }}
+              "roleDefinitionId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/', 'b24988ac-6180-42a0-ab88-20f7382dd24c')]",
```

Comment:
magic numbers! :tada:

Reply:
Yeah, not sure what to do here as these were given to me.  They are hard coded.